bundle/Resources/config/fieldtype_form_types.yml

```diff
@@ -4,13 +4,15 @@ parameters:
 services:
     netgen.ezforms.form.type.fieldtype.ezuser_create:
         class: Netgen\Bundle\EzFormsBundle\Form\Type\FieldType\UserCreateType
+        public: false
```

Comment:
Form types need to be public on Symfony 2.8.

My usual way is to comment out this line, to be uncommented when support for Symfony 2.8 stops.